psqlgraph/query.py

```diff
@@ -468,9 +465,15 @@ def prop_in(self, key, values):
             g.prop_in('key1', ['Yes', 'yes', 'True', 'true']).count()
 
         """
-
+        entity = self.entity()
+        col = entity._props
+        if is_list_prop(entity, key):
+            # applicable only to text arrays
+            # see https://www.postgresql.org/docs/9.4/functions-json.html
+            # has_any is `?|` under the hood and that requires the right operand to be a text array
+            return self.filter(col[key].has_any(array(values)))
```

Comment:
do you think we could combine `jsonb_array_elements_text` and also treat `values` elements as `str` as well? this might be a bit tricky with floats, but ints should work?

Reply:
I did try this, but sadly it raises an error when used as part of a column clause, its meant to be in from clauses as seen in the example from postgresql docs
``` sql
select * from json_array_elements_text('["foo", "bar"]')
```

I do think its something that can still be explored, Found a promising approach on [stackoverflow](https://dba.stackexchange.com/questions/54283/how-to-turn-json-array-into-postgres-array/54289) using subqueries. Just not very sure how nice it will work with the way we use the queries

Reply:
I see. lets get this in and do more research as part of the ticket you referenced